BioCrypt-Personal-V0.4 Source-code/main.py

```
@@ -0,0 +1,1814 @@
+                keys = {}
+
+            # Add the new CID and key to the dictionary
+            keys[cid] = key.hex()
+            print("Updated encryption keys:", keys)  # Debug
+
+            # Save the updated dictionary back to the file
+            with open(KEYS_FILE, "w") as f:
+                # Pretty-print JSON for readability
+                json.dump(keys, f, indent=4)
+            print("Encryption key saved successfully.")  # Debug
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)